.github/workflows/openvex.yml

```diff
@@ -0,0 +1,19 @@
+name: openvex
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  vexctl:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 #v4.1.1
+      - name: Set env
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: openvex/generate-vex@31b415924ea0d72ed5f2640f1dee59dea6c2770b
```

Comment:
Can we pin this to the latest major as well?

Reply:
I see `openvex` has not tagged any release yet. Can we pin this to the latest `c59881b41451d7ccba5c3b74cd195382b8971fcd` commit then?

Reply:
I've raised an issue regarding releases: https://github.com/openvex/generate-vex/issues/2.

Reply:
> I see `openvex` has not tagged any release yet. Can we pin this to the latest `c59881b41451d7ccba5c3b74cd195382b8971fcd` commit then?

Done.